### PR TITLE
Typos

### PR DESCRIPTION
**Pull request by flashonfire**
Typos and better wording in several places

### CODE REVIEW

File: apps/onyx/lib/screens/izly/pages/izly_page.dart
Comment (hatch01) [inline apps/onyx/lib/screens/izly/pages/izly_page.dart:1]: would be better if QrCode take an s only when there are multiple qrcodes remaining

Comment (hatch01) [inline apps/onyx/lib/screens/izly/pages/izly_page.dart:1]: would be better if QrCode take an s only when there are multiple qrcodes remaining

File: apps/onyx/lib/screens/izly/pages/izly_recharge_page.dart
Comment (hatch01) [inline apps/onyx/lib/screens/izly/pages/izly_recharge_page.dart:1]: Did you ensure that there is no overflow with such a long text?

**Comment by hatch01** [comment]
Thanks for your useful PR. Really appreciated. (yes, my French and my English are catastrophic)

File: apps/onyx/lib/screens/izly/pages/izly_recharge_page.dart
Comment (flashonfire) [reply apps/onyx/lib/screens/izly/pages/izly_recharge_page.dart:1]: Sure, it looks like this :
image

File: apps/onyx/lib/screens/izly/pages/izly_page.dart
Comment (flashonfire) [reply apps/onyx/lib/screens/izly/pages/izly_page.dart:1]: Good point, i'll make the changes tomorrow

Comment (flashonfire) [reply apps/onyx/lib/screens/izly/pages/izly_page.dart:1]: Well, i did it now

**Comment by flashonfire** [comment]
Should be ready to merge @hatch01

**Review by hatch01 — APPROVED** [review]

**Pull request merged**